### PR TITLE
feat(image-fetcher): Reformat parameters for flask server

### PR DESCRIPTION
**Pull request by cassand**

### CODE REVIEW

File: backend/app/modules/imagefetcher/FetcherActor.scala [hunk 1]
```diff
@@ -46,25 +46,27 @@ class FetcherActor(ws: WSClient, serverUrl: String) extends Actor {
...
 
     val initialParams = Seq(
-      ("start", message.start),
+      ("date", message.start),
       ("delta", message.delta),
       ("polygon", message.polygon.toString))
 
     val params =
-      if (message.scale.isDefined) initialParams :+ ("scale", message.scale.get.toString)
+      if (message.scale.isDefined) initialParams :+ ("scale", message.scale.get.toInt.toString)
```
Comment (ArchangelX360) [inline backend/app/modules/imagefetcher/FetcherActor.scala]: Python API match float also now

File: backend/app/modules/imagefetcher/FetcherActor.scala [hunk 2]
```diff
@@ -46,25 +46,27 @@ class FetcherActor(ws: WSClient, serverUrl: String) extends Actor {
...
       else initialParams
 
     val request = ws.url(serverUrl + "/rgb")
         .withQueryString(params:_*)
         .get()
 
+    val currenSender = sender
```
Comment (ArchangelX360) [inline backend/app/modules/imagefetcher/FetcherActor.scala]: Current not curren :)

**Review by ArchangelX360 — CHANGES_REQUESTED** [review]

File: backend/app/modules/imagefetcher/FetcherActor.scala [hunk 3]
```diff
@@ -58,13 +58,15 @@ class FetcherActor(ws: WSClient, serverUrl: String) extends Actor {
         .withQueryString(params:_*)
         .get()
 
+    val currentSender = sender
```
Comment (FunkySayu) [inline backend/app/modules/imagefetcher/FetcherActor.scala:61]: I don't get this change. Is there a scope issue?

Comment (cassand) [reply backend/app/modules/imagefetcher/FetcherActor.scala:61]: Yes, sender refer to the sender of the last message receive by the actor, therefore, access it from a future can lead to an error (access to the sender of the next message instead)

sender is a method / currentSender is a value

**Review by bouillonpi — DISMISSED** [review]

**Pull request merged**